server/src/database.rs

```diff
@@ -169,8 +169,6 @@ impl Database {
         index!("domains", "local_name");
         index!("domains", "remote_name");
         index!("domains", "timestamp");
-        index!("domains", "public_ip");
-        index!("domains", "email");
```

Comment:
we want to keep the index, just not UNIQUE ones. So the right fix is to create a different macro for unique vs. non unique indexes. Actually the timestamp one should not be unique either.